app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -89,16 +80,18 @@ class AboutActivity : BaseNavigationActivity() {
         }
     }
 
-    private fun _openEmailIntent() {
-        val emailIndent = Intent().apply {
-            action = Intent.ACTION_SEND
-            type = "plain/text"
-            putExtra(Intent.EXTRA_EMAIL, arrayOf(getString(R.string.contact_email_address)))
-        }
-        if (emailIndent.resolveActivity(packageManager) != null) {
-            startActivity(emailIndent)
-        } else {
-            Toast.makeText(this, R.string.no_email_app_available, Toast.LENGTH_SHORT).show()
+    private fun _selectContactConcern() {
+        val sendFeedback = ContactUtils(this)
```

Comment:
`sendFeedback` is not a proper variable name. Do you mean sth like `contactInterface`?